packages/flutter/lib/src/material/raised_button.dart

```diff
@@ -121,12 +131,17 @@ class RaisedButton extends StatelessWidget {
 
   /// Called by the underlying [InkWell] widget's [InkWell.onHighlightChanged]
   /// callback.
+  ///
+  /// This argument is ignored when adapting for iOS since [CupertinoButton]
```

Comment:
I suppose it's just me but all of these "this argument is ignored" qualifiers sound to me like we should have created a separate API for widgets that adapt to the target platform.